orca-bakery/src/main/groovy/com/netflix/spinnaker/orca/bakery/pipeline/BakeStage.groovy

```diff
@@ -104,7 +106,7 @@ class BakeStage implements StageDefinitionBuilder {
       stage.context.amiSuffix = clock.instant().atZone(UTC).format("yyyyMMddHHmmss")
     }
     return deployRegions.collect {
-      stage.context - ["regions": stage.context.regions] + ([
+      stage.context - ["regions": stage.context.regions, "skipRegionDetection": skipRegionDetection] + ([
```

Comment:
nice, thanks for changing this.

another issue, i just realized - i don't think it's a huge deal, but best to address before it causes weird issues.

```suggestion
      stage.context - ["regions": stage.context.regions, "skipRegionDetection": stage.context.skipRegionDetection] + ([
```

otherwise. if `stage.context.skipRegionDetection = 0` this subtraction won't work